### PR TITLE
llvmPackages_{12,13,14,15,16,17,18,git}.lldb: add patch to remove origin variable

### LINKED ISSUE #315214: liblldb.so.18.1 shared library dependencies not correctly set
### Describe the bug
```
$ ldd /nix/store/2033wdzw97yc2j0s51fa1q6max99i6a3-lldb-18.1.5-lib/lib/liblldb.so.18.1 | rg "=>" | awk '{print $3}' | xargs realpath
/usr/lib/libz.so.1.3.1
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/libpython3.11.so.1.0
/usr/lib/liblua5.3.so.5.3.6
/usr/lib/libpanelw.so.6.4
/usr/lib/libncursesw.so.6.4
/usr/lib/libformw.so.6.4
/usr/lib/libxml2.so.2.12.6
/usr/lib/libedit.so.0.0.72
/usr/lib/libdl.so.2
/nix/store/92g7nfqk2g4yzy56lfrla3w89i61vnkg-clang-18.1.5-lib/lib/libclang-cpp.so.18.1
/nix/store/xpw2cabpd7vg67gakv727rz0gz9k84dy-llvm-18.1.5-lib/lib/libLLVM.so.18.1
/usr/lib/libstdc++.so.6.0.32
/usr/lib/libm.so.6
/usr/lib/libgcc_s.so.1
/usr/lib/libc.so.6
/usr/lib/liblzma.so.5.6.1
/usr/lib/libicuuc.so.74.2
/nix/store/kbg5m7fyi1w23fyfmxjhhzcbd577rpg0-libffi-3.4.6/lib/libffi.so.8.1.4
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib/librt.so.1
/usr/lib/libicudata.so.74.2
```

I expect that these dependencies should resolve to /nix/store/ and not to /usr/lib/.

### Steps To Reproduce

FWIW, I am working on adding LLVM 18 support to bpftrace CI. (We are loving nix's role in our CI, btw).

I'm not sure if enough information is contained in the store path I shared above. But if not, I have a reproducer here: https://github.com/danobi/bpftrace/tree/nix_llvm18-nixpkgs . Just run `nix build`. You should see the following failure:

```
$ git rev-parse HEAD
4e8414c021d9d4ebd1c19106b87ab069b56a9742
$ nix build
$ ./result/bin/bpftrace
./result/bin/bpftrace: error while loading shared libraries: libicuuc.so.74: cannot open shared object file: No such file or directory
```
I discovered lldb is related through LD_DEBUG=all:

```
$ LD_DEBUG=all ./result/bin/bpftrace
[..]
   2726159:     file=libicuuc.so.74 [0];  needed by /nix/store/hcag2qn4njvps4wpy7fnjb20y80fynni-lldb-18.1.5-lib/lib/../../../../lib/libxml2.so.2 [0]
   2726159:     find library=libicuuc.so.74 [0]; searching
[..]
```

### Expected behavior
lldb.so dynamic dependencies to resolve into nix store.

### Notify maintainers

(apologies if I got the list wrong)

@dtzWill
@ericson2314
@lovek323
@qyliss
@RossComputerGuy
@rrbutani
@sternenseemann

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.8.7-arch1-2, Arch Linux, noversion, rolling`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.0`
 - nixpkgs: `/home/dxu/.nix-defexpr/channels/nixpkgs`

```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by RossComputerGuy**
## Description of changes

Fixes #315214

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by paparodeo — COMMENTED** [review]
the lines with ORIGIN should just get removed. maybe in a different patch `origin-rpath-remove.patch`

**Comment by RossComputerGuy** [comment]
> the lines with ORIGIN should just get remove

No, I based the patch on the one we have for LLVM (`pkgs/development/compilers/llvm/17/llvm/gnu-install-dirs.patch`).

**Comment by paparodeo** [comment]
> kgs/development/compilers/llvm/17/llvm/gnu-install-dirs.patch

yeah, that patch is wrong too -- in the llvm18 init it was fixed up a bit to remove $ORIGIN.

**Comment by RossComputerGuy** [comment]
> yeah, that patch is wrong too -- in the llvm18 init it was fixed up a bit to remove $ORIGIN.

Oh huh, I forgot I made that change when I made the LLVM 18 PR.

**Comment by paparodeo** [comment]
in general, it seems that anything using `$ORIGIN` that is exiting the derivation path should be removed. `$ORIGIN/../lib` is ok for binaries in `/nix/store/.../bin/`.  and maybe if there is some subdirectory, things might what `$ORIGIN/lib-subdir` for things in `lib` to reference `lib/lib-subdir`. but even many of those uses we add absolute RPATHS anyway so are mostly redundant.

**Comment by RossComputerGuy** [comment]
Alright, I removed `$ORIGIN`.

File: pkgs/development/compilers/llvm/12/lldb/gnu-install-dirs.patch [hunk 1]
```diff
@@ -89,3 +89,16 @@ index b5633e2..86e4738 100644
...
+diff --git a/source/API/CMakeLists.txt b/source/API/CMakeLists.txt
+index b5633e2..86e4738 100644
+--- a/source/API/CMakeLists.txt	2024-05-30 19:08:14.241656491 -0700
++++ b/source/API/CMakeLists.txt	2024-05-30 19:08:48.218219667 -0700
+@@ -118,7 +118,7 @@
+ # (LLVM_LINK_LLVM_DYLIB). Add an additional rpath $ORIGIN/../../../../lib so
+ # that _lldb.so can be loaded from Python.
+ if(LLDB_ENABLE_PYTHON AND (BUILD_SHARED_LIBS OR LLVM_LINK_LLVM_DYLIB) AND UNIX AND NOT APPLE)
+-  set_property(TARGET liblldb APPEND PROPERTY INSTALL_RPATH "\$ORIGIN/../../../../lib${LLVM_LIBDIR_SUFFIX}")
++  set_property(TARGET liblldb APPEND PROPERTY INSTALL_RPATH "${LLVM_LIBRARY_DIR}")
```
Comment (paparodeo) [inline pkgs/development/compilers/llvm/12/lldb/gnu-install-dirs.patch:101]:
```suggestion
+  # set_property(TARGET liblldb APPEND PROPERTY INSTALL_RPATH "${LLVM_LIBRARY_DIR}")
```

File: pkgs/development/compilers/llvm/common/lldb/gnu-install-dirs.patch [hunk 2]
```diff
@@ -47,3 +47,16 @@ index 7d48491ec89a..c04543585588 100644
...
+diff --git a/source/API/CMakeLists.txt b/source/API/CMakeLists.txt
+index 7d48491ec89a..c04543585588 100644
+--- a/source/API/CMakeLists.txt	2024-05-30 18:59:13.449269853 -0700
++++ b/source/API/CMakeLists.txt	2024-05-30 19:00:33.721858164 -0700
+@@ -120,7 +120,7 @@
+ # (LLVM_LINK_LLVM_DYLIB). Add an additional rpath $ORIGIN/../../../../lib so
+ # that _lldb.so can be loaded from Python.
+ if(LLDB_ENABLE_PYTHON AND (BUILD_SHARED_LIBS OR LLVM_LINK_LLVM_DYLIB) AND UNIX AND NOT APPLE)
+-  set_property(TARGET liblldb APPEND PROPERTY INSTALL_RPATH "\$ORIGIN/../../../../lib${LLVM_LIBDIR_SUFFIX}")
++  set_property(TARGET liblldb APPEND PROPERTY INSTALL_RPATH "${LLVM_LIBRARY_DIR}")
```
Comment (paparodeo) [inline pkgs/development/compilers/llvm/common/lldb/gnu-install-dirs.patch:59]:
```suggestion
+  # set_property(TARGET liblldb APPEND PROPERTY INSTALL_RPATH "${LLVM_LIBRARY_DIR}")
```

**Review by paparodeo — COMMENTED** [review]
the lines that have origin can just be removed (i commented them out only to not break the patch) as they are not required.

File: pkgs/development/compilers/llvm/12/lldb/gnu-install-dirs.patch (hunk 1 above)
Comment (RossComputerGuy) [reply pkgs/development/compilers/llvm/12/lldb/gnu-install-dirs.patch]: I removed the line.

**Review by paparodeo — APPROVED** [review]
LGTM!

**Comment by paparodeo** [comment]
this should get the `backport release-24.05` label too

**Comment by paparodeo** [comment]
rpaths for LLVM12-18. looks like LLVM13 is still using `$ORIGIN/../../../../lib` for some reason.

<details><summary>patchelf --print-rpath $x|xargs -d : -n 1 </summary>

```console
$ORIGIN/..//nix/store/cxdbc9gkk08flmgvy070qdmnq0zppkij-lldb-12.0.1-lib/lib
///nix/store/wn3s8rxc7zv00ibj9gsj8f353yj4849s-llvm-12.0.1-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib
/nix/store/qhfk7hvngljfi8bck3b7rskl69h5xk8g-clang-12.0.1-lib/lib

$ORIGIN/..//nix/store/aici40kvdhlb4pmz2hrv2w60aclnc73k-lldb-13.0.1-lib/lib
///nix/store/lz8sjwcrfrk1jqmsrdm6ikrra5824y8z-llvm-13.0.1-lib/lib
$ORIGIN/../../../../lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/d4dzkmwkyrkc1l8z9x7vcdj193fx4g45-gcc-12.3.0-lib/lib
/nix/store/fvgyp8fqjkxif8gbab23h6dvgqi55sd8-clang-13.0.1-lib/lib

$ORIGIN/..//nix/store/wfr0lw9g3dragcyg8rnv1i064ncvwj93-lldb-14.0.6-lib/lib
/nix/store/lh3dq18c0a6d8nybvm9z68fr04vlq8w1-llvm-14.0.6-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib
/nix/store/js44niygpsvdni002pjk7jx1yx4gr822-clang-14.0.6-lib/lib

$ORIGIN/..//nix/store/q8ypj9ymv55nv551nvhf7pirs9nw4yga-lldb-15.0.7-lib/lib
/nix/store/33drjrjajqv0xd7jg5hiq47pdwd641gh-llvm-15.0.7-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib
/nix/store/k8m2f5rwb4ryx20cm1mjy1gmrfjbksc1-clang-15.0.7-lib/lib

$ORIGIN/..//nix/store/0jcgihrrjxhdy9lw2mgpyq3lazy3ja2s-lldb-16.0.6-lib/lib
/nix/store/gshzcbn5677fyp2g46wx49r7jdzsa05h-llvm-16.0.6-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/px860gmf08bwx7cwdnilzscgp3540j8p-clang-16.0.6-lib/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib

$ORIGIN/..//nix/store/cxzf6ryar2ldvdvm9ly33fifcmf05hmg-lldb-17.0.6-lib/lib
/nix/store/ax9wbqknwawnl21kvkp8l6jn2r6q0l90-llvm-17.0.6-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/wxxgsgjxbnkkyczgf8lkbfrsqiywm8bi-clang-17.0.6-lib/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib

/nix/store/yvk6901c28z1vpshzvp0z4qyxi274jlh-llvm-18.1.6-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/5d52f93rlhp8xbjc94kycxfs70dbwna6-clang-18.1.6-lib/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib
```
</details>

**Comment by RossComputerGuy** [comment]
@paparodeo Fixed, LLVM 13 uses it's own separate patch for lldb like how LLVM 12 does.

**Review by paparodeo — APPROVED** [review]
LGTM

<details><summary>patchelf --print-rpath $x|xargs -d : -n 1</summary>

```console
$ORIGIN/..//nix/store/cxdbc9gkk08flmgvy070qdmnq0zppkij-lldb-12.0.1-lib/lib
///nix/store/wn3s8rxc7zv00ibj9gsj8f353yj4849s-llvm-12.0.1-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib
/nix/store/qhfk7hvngljfi8bck3b7rskl69h5xk8g-clang-12.0.1-lib/lib

$ORIGIN/..//nix/store/cgvrb3bd5z9f063kayyiska4fiq2sjg5-lldb-13.0.1-lib/lib
///nix/store/lz8sjwcrfrk1jqmsrdm6ikrra5824y8z-llvm-13.0.1-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/d4dzkmwkyrkc1l8z9x7vcdj193fx4g45-gcc-12.3.0-lib/lib
/nix/store/fvgyp8fqjkxif8gbab23h6dvgqi55sd8-clang-13.0.1-lib/lib

$ORIGIN/..//nix/store/wfr0lw9g3dragcyg8rnv1i064ncvwj93-lldb-14.0.6-lib/lib
/nix/store/lh3dq18c0a6d8nybvm9z68fr04vlq8w1-llvm-14.0.6-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib
/nix/store/js44niygpsvdni002pjk7jx1yx4gr822-clang-14.0.6-lib/lib

$ORIGIN/..//nix/store/q8ypj9ymv55nv551nvhf7pirs9nw4yga-lldb-15.0.7-lib/lib
/nix/store/33drjrjajqv0xd7jg5hiq47pdwd641gh-llvm-15.0.7-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib
/nix/store/k8m2f5rwb4ryx20cm1mjy1gmrfjbksc1-clang-15.0.7-lib/lib

$ORIGIN/..//nix/store/0jcgihrrjxhdy9lw2mgpyq3lazy3ja2s-lldb-16.0.6-lib/lib
/nix/store/gshzcbn5677fyp2g46wx49r7jdzsa05h-llvm-16.0.6-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/px860gmf08bwx7cwdnilzscgp3540j8p-clang-16.0.6-lib/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib

$ORIGIN/..//nix/store/cxzf6ryar2ldvdvm9ly33fifcmf05hmg-lldb-17.0.6-lib/lib
/nix/store/ax9wbqknwawnl21kvkp8l6jn2r6q0l90-llvm-17.0.6-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/wxxgsgjxbnkkyczgf8lkbfrsqiywm8bi-clang-17.0.6-lib/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib

/nix/store/yvk6901c28z1vpshzvp0z4qyxi274jlh-llvm-18.1.6-lib/lib
/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib
/nix/store/x5mpg1pjazhvja8aq0yxig2zgz51i9c1-lua-5.3.6/lib
/nix/store/j4m3lwhh4x0bzilvi77d512ahry775nq-ncurses-6.4/lib
/nix/store/lv6nackqis28gg7l2ic43f6nk52hb39g-zlib-1.3.1/lib
/nix/store/bmjckrv5ryxahvkldr47jyx7zmlrcan3-libedit-20230828-3.1/lib
/nix/store/kf7fzax0av9him9m4zcmbqmihh5108wq-libxml2-2.12.7/lib
/nix/store/5d52f93rlhp8xbjc94kycxfs70dbwna6-clang-18.1.6-lib/lib
/nix/store/k7zgvzp2r31zkg9xqgjim7mbknryv6bs-glibc-2.39-52/lib
/nix/store/xvzz97yk73hw03v5dhhz3j47ggwf1yq1-gcc-13.2.0-lib/lib
```
</details>

**Comment by danobi** [comment]
Gentle ping - would love to get this in so bpftrace can fix its pipelines

**Pull request merged**